tests/unit/script/htmlimageelement.rs

```diff
@@ -2,7 +2,90 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use script::test::DOMString;
+use script::test::sizes::parse_a_sizes_attribute;
 use script::test::srcset::{Descriptor, ImageSource, parse_a_srcset_attribute};
+use style::media_queries::{MediaCondition, MediaFeatureExpression};
+use style::servo::media_queries::{ExpressionKind, Range};
+use style::values::specified::{Length, NoCalcLength, AbsoluteLength};
+use style::values::specified::{source_size_list::SourceSizeList, source_size_list::SourceSize};
+
+
+#[test]
+fn no_default_provided() {
```

Comment:
I think I'd prefer to remove the unit tests and make the `SourceSizeList` and `SourceSize` fields private again, given there's a few tests that check all this in WPT: https://wpt.fyi/results/html/semantics/embedded-content/the-img-element/sizes

Reply:
I would prefer to retain the unit tests for the time being until we finish implementing all the pieces that are required in order for the WPT tests to actually work in Servo.

Reply:
That means that when I change SourceSizeList in Gecko I need to update all the tests, which is cumbersome given this is tested already there... :/

Reply:
Ok, that's a reasonable argument against these changes. @paavininanda Let's undo the changes that were made to support re-adding the unit tests, and remove the unit tests. This PR can just focus on adding the parse_a_sizes_attribute function without using it anywhere.